### ISSUE TITLE
Cannot customize with template

### ISSUE THREAD

**Issue by NoxInmortus** [rustic-games/jilu#12]
Hello,

I'm trying to use jilu for projects of my owns, but the default template does no suit my needs.
I've added the example template in my CHANGELOG.md :
```
<!--
Config(
  github: ( repo: "rustic-games/jilu" ),
  accept_types: ["feat", "fix", "perf"],
  type_headers: {
    "feat": "Features",
    "fix": "Bug Fixes",
    "perf": "Performance Improvements"
  }
)

Template(
# My Change Log

## Upcoming Changes

{% for change in unreleased.changes %}
- {{ change.description }} ([`{{ change.commit.short_id }}`])
{%- endfor %}
)
-->
```
But then using `jilu` or `jilu | sponge` does not produce any expected output, It only displays the example template (still in the jinja form as above).
If I remove the Template() part, it display something like that :
```
# Changelog

All notable changes to this project are documented in this file.

The format is based on [Keep a Changelog], and this project adheres to
[Semantic Versioning]. The file is auto-generated using [Conventional Commits].

[keep a changelog]: https://keepachangelog.com/en/1.0.0/
[semantic versioning]: https://semver.org/spec/v2.0.0.html
[conventional commits]: https://www.conventionalcommits.org/en/v1.0.0-beta.4/

## Overview

- [unreleased](#unreleased)

## _[Unreleased]_

_nothing new to show for… yet!_

<!-- [releases] -->

[unreleased]: https://github.com/NoxInmortus/mysmth/commits

<!-- [commits] -->


<!--
Config(
  github: ( repo: "NoxInmortus/mysmth" ),
  accept_types: ["feat", "fix", "perf"],
  type_headers: {
    "feat": "Features",
    "fix": "Bug Fixes",
    "perf": "Performance Improvements"
  }
)
-->
```
I don't understand what's missing there, any help would be appreciated.

**Comment by JeanMertz** [comment]
`jilu` expects the commits to conform to the Conventional Commit style.

Could it be that this is not the case in your project?

I just created a `CHANGELOG.md` file on a repository that only uses conventional commits and then added the example config to the file:

```markdown
<!--
Config(
  github: ( repo: "rustic-games/jilu" ),
  accept_types: ["feat", "fix", "perf"],
  type_headers: {
    "feat": "Features",
    "fix": "Bug Fixes",
    "perf": "Performance Improvements"
  }
)

Template(
# My Change Log

## Upcoming Changes

{% for change in unreleased.changes %}
- {{ change.description }} ([`{{ change.commit.short_id }}`])
{%- endfor %}
)
-->
```

After running `jilu`, I got the expected outcome:

```markdown
# My Change Log

## Upcoming Changes


- example commit title ([`1b329d8`])
- another commit ([`43f6356`])
<!--
Config(
  github: ( repo: "rustic-games/jilu" ),
  accept_types: ["feat", "fix", "perf"],
  type_headers: {
    "feat": "Features",
    "fix": "Bug Fixes",
    "perf": "Performance Improvements"
  }
)

Template(
# My Change Log

## Upcoming Changes

{% for change in unreleased.changes %}
- {{ change.description }} ([`{{ change.commit.short_id }}`])
{%- endfor %}
)
-->
```

I do agree that Jilu needs to be better at reporting if there's something in the repository preventing it from generating the change log output.

I'll close this issue, as there doesn't appear to be a bug here, but feel free to respond with more details on your specific use-case, so that we can pinpoint why it isn't working for your specific case, and discuss what we could do to improve errors/warnings produced by Jilu in such cases.

**Comment by NoxInmortus** [comment]
Hello @JeanMertz 
An example : 
```
me@work:[~/git/roles/role-zabbix]: jilu
# Changelog

All notable changes to this project are documented in this file.

The format is based on [Keep a Changelog], and this project adheres to
[Semantic Versioning]. The file is auto-generated using [Conventional Commits].

[keep a changelog]: https://keepachangelog.com/en/1.0.0/
[semantic versioning]: https://semver.org/spec/v2.0.0.html
[conventional commits]: https://www.conventionalcommits.org/en/v1.0.0-beta.4/

## Overview

- [unreleased](#unreleased)

## _[Unreleased]_

- fixes: Api task, psk parameters defaults ([`9f5ce6f`])
- docs: Update README ([`8f09744`])
- fixes: Zabbix Agent tls psksecret and api stuff ([`4a1abb1`])
- fixes: zombie code in api.yml task ([`aef5c2d`])
- test: api management ([`ea3b0c8`])
- fixes: zabbx proxy ([`1dccb85`])
- refactor: Moved included_vars, changed tags, fixed stuff on external tasks ([`1bc0483`])
- fixes: agent/proxy tests ([`d9fa3ae`])

<!-- [releases] -->

[unreleased]: #/commits

<!-- [commits] -->

[`9f5ce6f`]: #/commit/9f5ce6fd7df16196426ed8d256122adeded7138a
[`8f09744`]: #/commit/8f09744ae83d94d68330ee38e00169c0b39a2b01
[`4a1abb1`]: #/commit/4a1abb15d0053c3729dc354b34d240e2499e77dc
[`aef5c2d`]: #/commit/aef5c2dd6e4f24967433041e810a4fe35413317e
[`ea3b0c8`]: #/commit/ea3b0c8347a7403f3c8d613e3fa462e8abac2daa
[`1dccb85`]: #/commit/1dccb85bcfebc91342a0e791b0873e08ec319932
[`1bc0483`]: #/commit/1bc048309e2c148e24aa46c8b9b8f47193da5d80
[`d9fa3ae`]: #/commit/d9fa3ae9d01db3f9268b2031f804f3abada809cf

alban@alban-asus:[~/git/roles/role-zabbix]: vim CHANGELOG.md 
alban@alban-asus:[~/git/roles/role-zabbix]: jilu
# My Change Log

## Upcoming Changes


<!--
Config(
  github: ( repo: "rustic-games/jilu" ),
  accept_types: ["feat", "fix", "perf"],
  type_headers: {
    "feat": "Features",
    "fix": "Bug Fixes",
    "perf": "Performance Improvements"
  }
)

Template(
# My Change Log

## Upcoming Changes

{% for change in unreleased.changes %}
- {{ change.description }} ([`{{ change.commit.short_id }}`])
{%- endfor %}
)
-->
```
